Source/WebKit/NetworkProcess/storage/CacheStorageRecord.h

```diff
@@ -37,6 +37,7 @@ class CacheStorageRecordInformation {
     CacheStorageRecordInformation(NetworkCache::Key&&, double insertionTime, uint64_t identifier, uint64_t updateResponseCounter, uint64_t size, URL&&, bool hasVaryStar, HashMap<String, String>&& varyHeaders);
     void updateVaryHeaders(const WebCore::ResourceRequest&, const WebCore::ResourceResponse::CrossThreadData&);
     CacheStorageRecordInformation isolatedCopy() &&;
+    CacheStorageRecordInformation isolatedCopy() const &;
```

Comment:
We need to fix the root cause in crossThreadCopier(). Otherwise it is bound to bite us again. Finding the bug is great and the fix is good but I think we should address the foot-gun in the same PR. I don't think we need to minimize the patch size here since this will not be cherry-picked.

I worry we may fail to follow-up to address the root cause otherwise. Also, fixing `crossThreadCopier()` may identify other bugs that would be very valuable too.